docs/user-guide/environment-variables.md

```diff
@@ -81,6 +81,7 @@ These environment variables may or may not be available depending on what plugin
 | Name | Description |
 |------|-------|
 | SD_ARTIFACTS_DIR | Location of built/generated files. <br><br>**Note**: The `sd-teardown-screwdriver-artifact-bookend` step uploads artifacts from this directory into the Store unless the build is `ABORTED`. |
+| SD_META_DIR | Location of the [metadata](./metadata) file |
```

Comment:
can you word this in a way that it doesn't duplicate `SD_META_PATH` ? Perhaps call it `Folder path for`

Reply:
Ah, sorry. Bad copy-paste.